### ISSUE TITLE
✅ 🐞 Bug Report: PHP 8.2 dynamic property creation is deprecated in container layout. - [ED-12252]

### ISSUE THREAD

**Issue by arielruminski** [elementor/elementor#23830]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

After the PHP version was changed to 8.2, the site reports two deprecated functionality in the elementor\includes\elements\container.php file.

```
[18-Sep-2023 10:18:05 UTC] PHP Deprecated:  Creation of dynamic property Elementor\Includes\Elements\Container::$logical_dimensions_inline_start is deprecated in ...\app\plugins\elementor\includes\elements\container.php on line 42
[18-Sep-2023 10:18:05 UTC] PHP Deprecated:  Creation of dynamic property Elementor\Includes\Elements\Container::$logical_dimensions_inline_end is deprecated in ...\app\plugins\elementor\includes\elements\container.php on line 43
```

### Steps to reproduce

1. Enable the display of deprecated errors.
2. Change the layout to container on the subpage.
3. Set PHP version 8.2
4. Visit the subpage with the container layout.

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>Click to reveal</summary>

````txt


				
== Server Environment ==
	Operating System: WINNT
	Software: Apache/2.4.57 (Win64) PHP/8.0.29 mod_fcgid/2.3.10-dev
	MySQL version: mariadb.org binary distribution v10.6.5
	PHP Version: 8.2.8
	PHP Memory Limit: 128M
	PHP Max Input Vars: 2500
	PHP Max Post Size: 8M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.3.1
	WP Multisite: No
	Max Upload Size: 2 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
	Language: en-US
	Timezone: Europe/Copenhagen
	Debug Mode: Active

== Theme ==
	Name: FLL
	Version: 2.0.4
	Author: Web Solutions WEB team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: da_DK
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/117.0.0.0 Safari/537.36

== Active Plugins ==
	Advanced Custom Fields PRO
		Version: 6.2.1.1
		Author: WP Engine

	Category Order and Taxonomy Terms Order
		Version: 1.7.9
		Author: Nsp-Code

	Elementor
		Version: 3.16.2
		Author: Elementor.com

	Essential Addons for Elementor
		Version: 5.8.8
		Author: WPDeveloper

	Essential Addons for Elementor - Pro
		Version: 5.6.0
		Author: WPDeveloper

	First Lego League
		Version: 1.2.1
		Author: Web Solutions

	Loco Translate
		Version: 2.6.6
		Author: Tim Whitlock

	Login as User
		Version: 1.4.7
		Author: Web357

	Post Types Order
		Version: 2.1
		Author: Nsp Code

	Query Monitor
		Version: 3.13.1
		Author: John Blackbourn

	Regenerate Thumbnails
		Version: 3.1.6
		Author: Alex Mills (Viper007Bond)

	Slider Revolution
		Version: 6.6.16
		Author: ThemePunch

	WP Mail SMTP
		Version: 3.9.0
		Author: WP Mail SMTP

	Yoast SEO
		Version: 21.1
		Author: Team Yoast


== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active by default
	Grid Container: Inactive by default
	Editor Top Bar: Inactive by default
	Landing Pages: Active by default
	Nested Elements: Inactive by default
	Lazy Load Background Images: Inactive by default
	Global Style Guide: Inactive by default


== Log ==
	
Log: showing 20 of 392023-05-29 14:08:14 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.3',
  'to' => '3.13.4',
)]
2023-06-29 11:15:53 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.4',
  'to' => '3.14.1',
)]
2023-06-29 11:15:55 [info] elementor::elementor_updater Started 
2023-06-29 11:15:55 [info] Elementor/Upgrades - _on_each_version Start  
2023-06-29 11:15:55 [info] Elementor/Upgrades - _on_each_version Finished 
2023-06-29 11:15:55 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.4',
  'to' => '3.14.1',
)]
2023-09-06 10:33:00 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.14.1',
  'to' => '3.15.3',
)]
2023-09-06 10:33:01 [info] elementor::elementor_updater Started 
2023-09-06 10:33:01 [info] Elementor/Upgrades - _on_each_version Start  
2023-09-06 10:33:01 [info] Elementor/Upgrades - _on_each_version Finished 
2023-09-06 10:33:01 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.14.1',
  'to' => '3.15.3',
)]
2023-09-13 10:10:53 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.15.3',
  'to' => '3.16.2',
)]
2023-09-13 10:10:53 [info] elementor::elementor_updater Started 
2023-09-13 10:10:53 [info] Elementor/Upgrades - _on_each_version Start  
2023-09-13 10:10:53 [info] Elementor/Upgrades - _on_each_version Finished 
2023-09-13 10:10:53 [info] Elementor/Upgrades - _v_3_16_0_container_updates Start  
2023-09-13 10:10:53 [info] Elementor/Upgrades - _v_3_16_0_container_updates Finished 
2023-09-13 10:10:53 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Start  
2023-09-13 10:10:53 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Finished 
2023-09-13 10:10:53 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.15.3',
  'to' => '3.16.2',
)]

JS: showing 1 of 1JS: 2023-07-11 12:32:20 [error X 3][http://new.firstlegoleague.local/app/plugins/elementor/assets/js/editor.js?ver=3.14.1:29641:79] Cannot read properties of null (reading 'id') 

PHP: showing 2 of 2PHP: 2023-07-12 13:26:29 [warning X 2][..\plugins\elementor\modules\history\revisions-manager.php::363] Trying to access array offset on value of type null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-09-18 12:18:07 [notice X 3][..\plugins\elementor\includes\elements\container.php::43] Creation of dynamic property Elementor\Includes\Elements\Container::$logical_dimensions_inline_end is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]



== Elementor - Compatibility Tag ==
	
	Essential Addons for Elementor: Compatibility not specified
	Essential Addons for Elementor - Pro: Compatibility not specified
				

````

</details>

**Comment by artkrsk** [comment]
I do confirm the same issue on PHP 8.2.10

**Comment by nicholaszein** [comment]
Hello @arielruminski! 

Thank you for submitting your issue! 🙏 

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Best regards

**Comment by hdrhkhn** [comment]
Hi, I get the same error like this. How can i fix it ?

**Comment by towfiq1997** [comment]
Same error facing in Php 8.2 Version please have a look

**Comment by mahabub775** [comment]
Facing Same Problem in php  8.2.4 version,  has any solution ?

**Comment by arielruminski** [comment]
@nicholaszein has [prepared a fix](https://github.com/elementor/elementor/pull/23883), but it has not yet been released.

**Comment by ErikRobles** [comment]
Today is November 2023 and I can that this issue is still occurring. Any news?

**Comment by jeffosico** [comment]
November 6th, 2023 issue still occurring, need fix please!

**Comment by Coriolan-Bataille** [comment]
Any schedule release? or Should we consider to downgrade?

**Comment by eSOL101** [comment]
Any update on this? I cant access my wordpress admin or site.

**Comment by zahangir060** [comment]
Any update?

**Comment by Leadsquad** [comment]
Update?

**Comment by JoeD101** [comment]
For anyone still having problems, just sharing my issue appears to have been with Elementor Pro not being on the latest version. Once I deactivated or updated Elementor Pro (license was not active, causing it not to update) the warnings and problems went away. I was mistakenly only paying attention to Elementor being latest.

**Comment by nicholaszein** [comment]
Hello everyone!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor  v3.17.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
